_includes/header.html

```diff
@@ -22,15 +22,20 @@
       <path class="logo-mark" d="m99.1 28.6-24.7-24.7-22.9 22.9-22.8-22.9-24.7 24.7 22.9 22.9-22.9 22.8 24.7 24.7 22.9-22.9 22.8 22.9 24.7-24.7-22.9-22.9zm-11.4 45.7-13.3 13.3-17.2-17.1 13.3-13.3zm-30.5-41.9 17.2-17.2 13.3 13.3-17.1 17.2zm-19 19-22.8-22.8 13.3-13.3 36.2 36.2-36.2 36.1-13.3-13.3z"/>
     </svg>
   </div>
-  <crds-shared-header env="{{ site.crds_env | default: 'prod' }}" root-url="https://rockdev.crossroads.net"></crds-shared-header>
+  {% if site.crds_env == 'prod' %}
+    {% assign rock_url = 'my' %}
+  {% else %}
+    {% assign rock_url = 'mydev' %}
+  {% endif %}
+  <crds-shared-header env="{{ site.crds_env | default: 'prod' }}" root-url="https://{{ rock_url }}.crossroads.net"></crds-shared-header>
```

Comment:
Wouldn't we just be able to do `root-url=https://{{ site.CRDS_ROCK_ENDPOINT }}`?